.github/workflows/json-schema-validation.yml

```diff
@@ -0,0 +1,22 @@
+name: JSON Schema Validation
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm install
+      - run: npm run test ./schemas
```

Comment:
If I'm not mistaken, the 3 yml files in this folder are exactly the same except for this one that runs the test.

I suggest we create 4 jobs in one file:
- `build` (all the steps until this one)
- run JSON schema validation tests (depends on `build`)
- run  DIDs JSON-LD Validation tests (depends on `build`)
- run  DIDs JSON Schema Validation tests (depends on `build`)

This way we would only run `npm install` once and CI actions will be more readable

Reply:
I agree, I'd rather address in a subsequent PR... https://github.com/w3c/did-core-registry/issues/15